### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by protoknight12** [TodePond/CellPond#190] [Help] [answered]
So, when I tried to make an element with these properties:
https://cdn.discordapp.com/attachments/717737131665260556/965588237500436550/Problem.png
and the grid in which I can put the element squares just keeps resizing
@TodePond So, what should I do

**Comment by TodePond** [comment]
Hi! :)

Do you mean like this? 
![image](https://user-images.githubusercontent.com/15892272/163808533-70ed95c4-458f-422d-aa2c-a65dce778bd8.png)

It seems to work as intended I think for me? What do you want to happen? :)

**Comment by protoknight12** [comment]
I actually made a mistake

**Reply by protoknight12** [reply]
![CellPond - Google Chrome 4_18_2022 3_37_25 PM](https://user-images.githubusercontent.com/84569377/163809465-7b21e385-61da-4212-88fc-2eeec512457f.png)

**Reply by protoknight12** [reply]
You see how there is one missing grid cell in the bottom of the second image in the rules

**Comment by TodePond** [comment, marked as answer]
Ah ok yes I see! So, the "left" side of the rule is always the same shape as the "right" side of the rule. At the moment, you need to add another cell to the "left" to make a grid cell on the right.

Before:
![image](https://user-images.githubusercontent.com/15892272/163810164-ce3b2de7-0fe2-4d9d-a434-06c8241016db.png)

After:
![image](https://user-images.githubusercontent.com/15892272/163810191-690cfd1b-ae5b-456e-b1f6-56079cc4c2e8.png)

If you want the cell on the left to be "any colour", you can take out the red, green and blue parts of the colour: 
![image](https://user-images.githubusercontent.com/15892272/163810359-62ca9fd2-c98e-4be7-82b0-07df66e5419d.png)

![image](https://user-images.githubusercontent.com/15892272/163810484-db539512-c41d-45dc-a55b-ccb40c556e56.png)

Does that help at all? :)

**Reply by protoknight12** [reply]
yes, it helps a ton

**Reply by TodePond** [reply]
Brilliant! I know it works a bit weirdly at the moment - it's something I'm going to improve in the future :)

**Reply by protoknight12** [reply]
yeah, i suggest that you make a prebuilt 3x3 grid which you can extend and retract

**Reply by i-rember** [reply]
That sounds like a good idea.
